docs/whats-new.rst

```diff
@@ -4,6 +4,12 @@ What's New
 v0.1.4 (? ? 2015)
 --------------------
 
+Added topography.
+
+Added automated linear stability analysis.
+
+Added multi layered model subclass. 
+
```

Comment:
This doesn't seem complete. What about?
* the bug fix related to the sign of advection terms
* the new diagnostics
* the changes to sqg model
* the new logger

Anything else I'm missing?